extension/src/openvic-extension/utility/UITools.cpp

```diff
@@ -123,6 +128,51 @@ static Array get_events_from_shortcut_key(String const& key) {
 	return events;
 }
 
+Error try_create_shortcut_action_for_button(GUIButton* gui_button, String const& shortcut_key_name, String const& shortcut_hotkey_name = "") {
+	Array event_array = get_events_from_shortcut_key(shortcut_key_name);
+
+	ERR_FAIL_COND_V_MSG(
+		shortcut_key_name.length() != 0 && event_array.size() == 0, ERR_INVALID_PARAMETER,
+		vformat("Unknown shortcut key '%s' for GUI button %s", shortcut_key_name, gui_button->get_name())
+	);
+
+	if (shortcut_key_name.length() == 0) {
```

Comment:
```suggestion
	if (shortcut_key_name.is_empty()) {
```
Could use `is_empty` here and at the beginning of `get_events_from_shortcut_key`, and `!shortcut_key_name.is_empty()` in the error check above